src/main/java/uk/nhs/hee/trainee/details/api/CctResource.java

```diff
@@ -154,4 +160,28 @@ public ResponseEntity<CctCalculationDetailDto> createCalculation(
     return ResponseEntity.created(URI.create("/api/cct/calculation/" + savedCalculation.id()))
         .body(savedCalculation);
   }
+
+  /**
+   * Move all CCT calculations from one trainee to another.
+   *
+   * @param fromTraineeId The TIS ID of the trainee to move calculations from.
+   * @param toTraineeId   The TIS ID of the trainee to move calculations to.
+   * @return True if the calculations were moved, bad request if the target trainee does not exist.
+   */
+  @PatchMapping("/move/{fromTraineeId}/to/{toTraineeId}")
```

Comment:
Not sure about the api path, but I don't have a better suggestion right now so it'll do 😄 

I do think we could improve the response code if we can make a clear differentiation between the following scenarios:

| From                      | To             | Response |
| -------------------- | ----------- | ---------- |
| Not Exists              | Any           | 404          |
| Exists, No Actions  | Any           | 200          |
| Exists, Has Actions | Not Exists | 400          |
| Exists, Has Actions | Exists        | 200          |

Reply:
Ok, I'll raise PRs to tweak them all to return a count instead of a boolean; that way the final public endpoint can return a bit of a summary of everything that has moved.

Reply:
The response looks to need to be in the format `{<typeOfObject>: <count>, ...}` since some calls make multiple moves (e.g. formR partA and partB counts, LTFT forms and submission history items, etc.)